src/main/java/gregtech/api/mui/widget/GhostCircuitSlotWidget.java

```diff
@@ -135,8 +135,10 @@ private IPanelHandler getSelectorPanel() {
         return this.selectorPanel;
     }
 
-    private static class GhostCircuitSyncHandler extends ItemSlotSH {
+    private static class GhostCircuitSyncHandler extends PhantomItemSlotSH {
```

Comment:
Why was this changed to extend the Phantom Item Slot SH? We don't really need anything from a phantom item slot for our ghost circuits correct?